app/assets/stylesheets/custom.scss

```diff
@@ -268,12 +277,12 @@ ul, menu, dir {
 #networks {
   width: 100%;
   height: 100%;
-  padding-top: 200px;
+  margin-top: 200px;
 }
 
 #networks i {
   font-size: 60px;
-  color: #d1ec53;
+  color: #fbf8ed;
```

Comment:
Color literals like `#fbf8ed` should only be used in variable declarations; they should be referred to via variable everywhere else.